pkg/build/api/types.go

```diff
@@ -142,6 +146,9 @@ const (
 
 	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
 	BuildPhaseCancelled BuildPhase = "Cancelled"
+
+	// BuildPhaseSkipped indicates that a queued build was skipped.
+	BuildPhaseSkipped BuildPhase = "Skipped"
```

Comment:
@smarterclayton k, so I will proceed with 'canceled' instead of adding new phase... perhaps we can describe the reason of build being canceled in annotation. Or introduce new field "skipped" that will be set to true if the build was skipped (phase will be cancelled).